pyvista/plotting/widgets.py

```diff
@@ -1882,7 +1882,23 @@ def add_checkbox_button_widget(
 
         background_color : color_like, optional
             The background color of the button. Default is ``'white'``.
-
+        
+        Example
+        -------
+        The following example generate a static image of the widget.
+        
+        >>> import pyvista as pv
+        >>> mesh = pv.Sphere()
+        >>> p = pv.Plotter()
+        >>> actor = p.add_mesh(mesh)
+        >>> def toggle_vis(flag):
+        >>>    actor.SetVisibility(flag)
+        >>> p.add_checkbox_button_widget(toggle_vis, value=True)
+        >>> p.show()
+        
+        Download the interactive example at :ref: `_checkbox_widget_example`
+        
+        
```

Comment:
```suggestion
        >>> import pyvista as pv
        >>> mesh = pv.Sphere()
        >>> pl = pv.Plotter()
        >>> actor = pl.add_mesh(mesh)
        >>> def toggle_vis(flag):
        ...     actor.SetVisibility(flag)
        >>> pl.add_checkbox_button_widget(toggle_vis, value=True)
        >>> pl.show()

        Download the interactive example at :ref: `checkbox_widget_example`.

```

Reply:
Will the ref work with the space between `:ref:` and the rest?

Reply:
It won't. Good catch.